### DISCUSSION TITLE
Multiple instances to same MQTT Broker

### DISCUSSION THREAD

**Discussion by tommyguuuun** [Koenkk/zigbee2mqtt#19451] [General]
Hey guys,

i‘m currently using 2 pi4 with zigbee2mqtt on two floors to extend my zigbee network. I‘m using Homeassistant with the addon.

In the first instance i specified the server, user and passwort under the mqtt: and everything works fine.

but when i try to do the same on the second instance (using a different base_topic) all devices from both instances are kicked out of the broker.

Am i missing any additional setting besides changing the base_topic?

**Comment by francisp2** [comment]
Try a different user.

**Reply by tommyguuuun** [reply]
Sorry forgot to mention. Both instances are using different users already.

**Reply by francisp2** [reply]
Different client id ?

https://github.com/Koenkk/zigbee2mqtt/issues/148

**Reply by tommyguuuun** [reply]
Just tried to set one manually - same results :(

**Comment by tommyguuuun** [comment]
As it looks like nobody could help me, i just configured HA Remote for the moment... but it lacks some possibilities for my radiator :(

**Comment by 0verEngineer** [comment]
@Koenkk Can you comment on this? - I am planning to do this in the near future.

**Comment by mundschenk-at** [comment]
Changing the base topic is important, broker credentials can be identical. This sounds more like the broker is configured weirdly. Which broker are you using?
